modules/installation-osp-api-octavia.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * networking/openstack/load-balancing-openstack.adoc
+
+[id="installation-osp-api-octavia_{context}"]
+= Scaling clusters for application traffic by using Octavia
+
+{product-title} clusters that run on {rh-openstack-first} can use the Octavia
+load balancing service to distribute traffic across multiple VMs or floating IP
```

Comment:
This is purely a suggestion since it's the first use of VM in this module, but it's inconsistently spelled out throughout the docs. 

s/VM/virtual machine (VM)